release-notes/opensearch-security.release-notes-1.3.2.0.md

```diff
@@ -0,0 +1,14 @@
+## 2022-03-30 Version 1.3.2.0
+
```

Comment:
There are a few commits that are missed here namely #1716 #1719 #1733 #1725. Should they be added?
https://github.com/opensearch-project/security/commits/1.3

Reply:
Thanks for double checking, I'll have an updated version in a bit